lib/ecto/adapters/sql/sandbox.ex

```diff
@@ -27,6 +27,41 @@ defmodule Ecto.Adapters.SQL.Sandbox do
     GenServer.start_link(__MODULE__, {conn_mod, opts}, [name: name])
   end
 
+  def open_transaction(_pool, _worker, _timeout) do
+    raise "#{inspect __MODULE__}.open_transaction/3 should never be called"
```

Comment:
Let's improve the error message here a tiny bit. Something like: If you want to use ownership with sql sandbox, configure your repo like this and then call ownership_checkout passing the sandbox as second option.